hassio/src/addon-store/hassio-addon-repository.ts

```diff
@@ -16,14 +17,25 @@ import {
   HassioAddonRepository,
 } from "../../../src/data/hassio";
 import { navigate } from "../../../src/common/navigate";
+import { filterAndSort } from "../components/hassio-filter-and-sort";
 
 class HassioAddonRepositoryEl extends LitElement {
   @property() public hass!: HomeAssistant;
   @property() public repo!: HassioAddonRepository;
   @property() public addons!: HassioAddonInfo[];
+  @property() public filter!: string;
 
   protected render(): TemplateResult | void {
     const repo = this.repo;
+    const addons = memoizeOne(filterAndSort(this.addons, this.filter));
```

Comment:
By calling `memoizeOne` inside render, you are creating a new instance, and so nothing is memoized. You need to store the memoized function as an instance variable like this: https://github.com/home-assistant/home-assistant-polymer/blob/dev/src/layouts/hass-router-page.ts#L55-L66